lib/versions_repository.py

```diff
@@ -25,3 +25,18 @@ def setup_versions_repository(config):
         raise
 
     return versions_repo
+
+def read_version_and_milestone(versions_repo):
+    """
+    Read current version and milestone (alpha or beta) from VERSION file
+
+    Args:
+        versions_repo (GitRepository): packages metadata git repository
+    """
+    version_path = os.path.join(versions_repo.name, 'VERSION')
```

Comment:
Use `versions_repo.working_tree_dir` instead of name. This will make the function executable from any directory.
I'd rename the variable to `version_file_path`.